libraries/splunk_app.rb

```diff
@@ -69,7 +69,7 @@ def apps_dir(arg = nil)
 
       def local(arg = nil)
         val = set_or_return(:local, arg, kind_of: [TrueClass, FalseClass])
-        (url.nil? || url.empty?) ? val : true
+        url.nil? || url.empty? ? val : true
```

Comment:
Rubocop 0.42.0 was released on July 25th that started complaining about these syntax.